src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -56,39 +56,76 @@ async def _process_one_file(
 
     output_path.parent.mkdir(parents=True, exist_ok=True)
 
+    output_code = transpile_result.transpiled_code
+
+    if any(err.kind == ErrorKind.PARSING for err in error_list):
```

Comment:
How do we handle ErrorKind.Internal?  the example of Recursive CTE?

Reply:
That is a good question. The `if any(...)` above detects cases where the output code is known to be incomplete (due to parse errors), which would therefore be useless to the user. In which case we display the input unchanged instead. 

Other kinds of errors are dealt with the "regular" way, with the output displayed under the header. Are there other cases (apart from parsing errors) where we should display the input?

Reply:
I would (somewhat hypocritically) convert them to something like "Your source code relies on features not yet supported by Lakebridge. Contact support for help."